Sources/SmileID/Classes/Camera/CameraManager.swift

```diff
@@ -179,3 +203,17 @@ extension CameraManager: AVCaptureVideoDataOutputSampleBufferDelegate {
         self.sampleBuffer = imageBuffer
     }
 }
+
+extension CameraManager: AVCapturePhotoCaptureDelegate {
+    func photoOutput(_ output: AVCapturePhotoOutput, didFinishProcessingPhoto photo: AVCapturePhoto, error: Error?) {
+        if let error {
+            return
+        }
+        if let imageData = photo.fileDataRepresentation(), let image = UIImage(data: imageData) {
+            self.capturedImage  = image
+        } else {
+            print(error?.localizedDescription)
+            return
```

Comment:
same question here: how would this bubble up to the consumer? how would we proceed forward from here?